apps/examples/fs_test/smartfs_test.c

```diff
@@ -0,0 +1,337 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/time.h>
+#include <fcntl.h>
+#include <errno.h>
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define USAGE                                                           \
+	"\n FS R/W Performance Test\n"                                      \
+	"\n Usage :     smartfs_test <operation> <path> <size> <iteration> <print_option>\n"  \
+	"\nOperation :  1.Read/Write / 2.Read / 3.Write / 4. Write/Remove \n"                  \
+	" Path :        Test File Path\n"                                   \
+	" Size :        Write/Read Data size\n"                             \
+	" Iteration :   Number of repetitions\n"                            \
+	" Detail :      Enable(y) or Disable(n) Printing Details Option\n"  \
+	"               (Printing is affects at R/W Performance)\n"         \
+	"\n ex) smartfs_test 1 /mnt/test 4096 30 y\n"
+
+#define FS_RW_ITERATION_MAX 10000
+
+enum TEST_OPTS {
+	TEST_OPT_WRITEREAD = 1,
+	TEST_OPT_READ = 2,
+	TEST_OPT_WRITE = 3,
+	TEST_OPT_WRITEREMOVE = 4,
+	TEST_OPT_MAX,
+};
+
+/****************************************************************************
+ * Global Variables
+ ****************************************************************************/
+static int g_total_time;
+char g_printopt;
+
+/****************************************************************************
+ * smartfs_test
+ ****************************************************************************/
+
+static inline long get_time(void)
+{
+	struct timeval tv;
+
+	gettimeofday(&tv, NULL);
+	return (tv.tv_sec * 1000 + tv.tv_usec / 1000);
```

Comment:
Is this intended result?
If `tv.tv_sec` is 1, and `tv.tv_usec` is 10, it means the result of `gettimeofday` is 1s10us.
But with your calculation, the result of `get_time()` indicates 1000s 0.001us.
I don't know why you need this time.